### ISSUE TITLE
Разместить коды программы в репозитории

### ISSUE THREAD

**Issue by wldmr-key** [DenisTR-KL/Kurasch-po-BD-2020#2]
Надеюсь, хоть что-то уже есть?

**Comment by DenisTR-KL** [comment]
Разместил в репозиторий, окно входа в программу . 

Сейчас оно работает в связи с локальным сервером MAMP. 
Нормально контактирует с ним(сервером), не существующие логины и не правильные пароли отклонят, с правильными соглашается.

Но я ещё хочу попробовать связать его(да и всю прогу) с локальным сервером на постгресс(который pgAdmin4) или используя инструменты Visual Studio,  встроенную локальную БД студии.

**Comment by wldmr-key** [comment]
да, можешь и MAMP оставить - для целей курсовой этого в принципе достаточно. Времени уже нет
